packages/components/inputs/localized-rich-text-input/src/localized-rich-text-input.tsx

```diff
@@ -181,160 +195,183 @@ const expandedTranslationsReducer = (
 // can get quite confusing. We try to stick to expand/collapse for the
 // multiline inputs, while we use show/hide/open/close for the remaining
 // languages.
-const LocalizedRichTextInput = (props: TLocalizedRichTextInputProps) => {
-  if (!props.isReadOnly) {
-    warning(
-      typeof props.onChange === 'function',
-      'LocalizedRichTextInput: `onChange` is required when input is not read only.'
+const LocalizedRichTextInput: ForwardRefExoticComponent<
+  TLocalizedRichTextInputProps & RefAttributes<unknown>
+> &
+  Partial<StaticProps> = forwardRef(
+  (props: TLocalizedRichTextInputProps, ref) => {
+    if (!props.isReadOnly) {
+      warning(
+        typeof props.onChange === 'function',
+        'LocalizedRichTextInput: `onChange` is required when input is not read only.'
+      );
+    }
+
+    if (props.showExpandIcon) {
+      warning(
+        typeof props.onClickExpand === 'function',
+        'LocalizedRichTextInput: "onClickExpand" is required when showExpandIcon is true'
+      );
+    }
+
+    if (props.hideLanguageExpansionControls) {
+      warning(
+        typeof props.defaultExpandLanguages !== 'boolean',
+        'LocalizedRichTextInput: "defaultExpandLanguages" does not have any effect when "hideLanguageExpansionControls" is set.'
+      );
+    }
+
+    const initialExpandedTranslationsState = Object.keys(props.value).reduce(
+      (translations, locale) => {
+        return {
+          [locale]: Boolean(props.defaultExpandMultilineText),
+          ...translations,
+        };
+      },
+      {}
     );
-  }
 
-  if (props.showExpandIcon) {
-    warning(
-      typeof props.onClickExpand === 'function',
-      'LocalizedRichTextInput: "onClickExpand" is required when showExpandIcon is true'
+    const [expandedTranslationsState, expandedTranslationsDispatch] =
+      useReducer(expandedTranslationsReducer, initialExpandedTranslationsState);
+
+    const defaultExpansionState =
+      props.hideLanguageExpansionControls ||
+      props.defaultExpandLanguages ||
+      // useToggleState's default is `true`, but we want `false`
+      false;
```

Comment:
nit: If we get to this point of the condition is because both `hideLanguageExpansionControls` and `defaultExpandLanguages` are falsy values, right?

Maybe this is because they might be `undefined` and if you pass that value to `useToggleState` it will use `true`, which we don't want.

If that's the case, perhaps we could use something like this:
```
const defaultExpansionState = Boolean(
      props.hideLanguageExpansionControls ||
      props.defaultExpandLanguages 
);
```